release-please-config.json

```diff
@@ -0,0 +1,76 @@
+{
+  "separate-pull-requests": true,
```

Comment:
@alxckn this setting configures release please to open separate prs for each artifact.